portfolio/src/main/webapp/index.html

```diff
@@ -51,9 +49,17 @@ <h2>Secret fact</h2>
       
       <h2>Comments</h2>
         <form action="/data" method="POST">
-            <p>Insert comments below:</p>
-            <input type="text" name="text">
+          <p>Insert comments below:</p>
+          <p>Name: <p>
```

Comment:
Is the space after "Name: " intentional?

Reply:
No it is not.